pkg/types.go

```diff
@@ -19,10 +19,10 @@ const (
 	SonobuoyLabelComponentName     = "component"
 	SonobuoyLabelComponentValue    = "sonobuoy"
 	DefaultToolsRepository         = "quay.io/opct"
-	ControllerImage                = "quay.io/opct/opct:v0.6.0-alpha.1"
-	PluginsImage                   = "plugin-openshift-tests:v0.6.0-alpha.1"
-	CollectorImage                 = "plugin-artifacts-collector:v0.6.0-alpha.1"
-	MustGatherMonitoringImage      = "must-gather-monitoring:v0.6.0-alpha.1"
+	ControllerImage                = "quay.io/opct/opct:v0.6.0-alpha.2"
+	PluginsImage                   = "plugin-openshift-tests:v0.6.0-alpha.2"
+	CollectorImage                 = "plugin-artifacts-collector:v0.6.0-alpha.2"
+	MustGatherMonitoringImage      = "must-gather-monitoring:v0.6.0-alpha.2"
```

Comment:
I believe we can bump to RC to test all together, so we can use the CI jobs to validate the pre-v0.6 and wrap up this quicker than validating individual changes.

```suggestion
	ControllerImage                = "quay.io/opct/opct:v0.6.0-rc.0"
	PluginsImage                   = "plugin-openshift-tests:v0.6.0-rc.0"
	CollectorImage                 = "plugin-artifacts-collector:v0.6.0-rc.0"
	MustGatherMonitoringImage      = "must-gather-monitoring:v0.6.0-rc.0"
```